Assets/MRTK/SDK/Features/UX/Interactable/Prefabs/RadialSet.prefab

```diff
@@ -986,8 +986,8 @@ MonoBehaviour:
   distanceSpaceMode: 1
   startPushDistance: -0.01
   maxPushDistance: 0.2
-  pressDistance: -0.004
-  releaseDistanceDelta: 0.0008
+  pressDistance: -0.002
+  releaseDistanceDelta: 0.006
```

Comment:
Do we need to change the default press distance for the radial prefab not just the radial set?

Reply:
Ah it looks like the Radial prefab just has an Interactable attached and not an Interactable and Pressable button like the buttons in the Radial Set prefab.